api/CloseEvent.json

```diff
@@ -0,0 +1,147 @@
+{
+  "api": {
+    "CloseEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CloseEvent",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "8",
```

Comment:
The note implies this should be some version before 8 or, if unknown, `true`

Reply:
I think it's correct.

Prior to Firefox 8, Firefox did not support the `CloseEvent` as defined by the spec. Rather, all listeners registered via `WebSocket.onclose()` were notified with an unspecific event, presumably without the properties of a real `CloseEvent`.

One could argue though that this note would be better placed in `WebSocket/onclose`.

Reply:
Ah, thank you for the explanation. I should've read it more closely—it makes sense your way.